src/main/java/seedu/address/logic/parser/deliverable/AddCommandParser.java

```diff
@@ -33,15 +33,20 @@ public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_TITLE, PREFIX_DESCRIPTION, PREFIX_DEADLINE, PREFIX_CONTACTS);
 
-        if (arePrefixesAbsent(argMultimap, PREFIX_TITLE)
-                || !argMultimap.getPreamble().isEmpty()) {
+        if (!arePrefixesPresent(argMultimap, PREFIX_TITLE) || !argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
         }
 
         Title title = ParserUtil.parseTitle(argMultimap.getValue(PREFIX_TITLE).get());
-        Description description = ParserUtil.parseDescription(argMultimap.getValue(PREFIX_DESCRIPTION).get());
-        Deadline deadline = ParserUtil.parseDeadline(argMultimap.getValue(PREFIX_DEADLINE).get());
-        String contacts = ParserUtil.parseContacts(argMultimap.getValue(PREFIX_CONTACTS).get());
+        Description description = argMultimap.getValue(PREFIX_DESCRIPTION).isEmpty()
+                ? Description.createEmptyDescription()
+                : ParserUtil.parseDescription(argMultimap.getValue(PREFIX_DESCRIPTION).get());
+        Deadline deadline = argMultimap.getValue(PREFIX_DEADLINE).isEmpty()
+                ? Deadline.createEmptyDeadline()
+                : ParserUtil.parseDeadline(argMultimap.getValue(PREFIX_DEADLINE).get());
+        String contacts = argMultimap.getValue(PREFIX_CONTACTS).isEmpty()
+                ? "NIL"
+                : ParserUtil.parseContacts(argMultimap.getValue(PREFIX_CONTACTS).get());
```

Comment:
Perhaps the code can be reduced by using Optional.orElse("NIL") instead of using the conditional operators